pyproject.toml

```diff
@@ -131,6 +129,9 @@ ignore = [
     # flake8-fixme (FIX)
     "FIX002",  # Line contains TODO, consider resolving the issue.
 
+    # flake8-implicit-str-concat (ISC)
+    "ISC001",  # Implicitly concatenate string literals on one line.
```

Comment:
Ignoring this specific rule, as recommended by `ruff format` i.e.,

```bash
warning: The following rules may cause conflicts when used with the formatter: `ISC001`. To avoid unexpected behavior, we recommend disabling these rules, either by removing them from the `select` or `extend-select` configuration, or adding them to the `ignore` configuration.
```